### ISSUE TITLE
Grid System dose not work

### ISSUE THREAD

**Issue by yunhao** [CosmicMind/Material#613]
Material 2.3.19
Xcode8

I tried to follow the tutorials in [http://www.cosmicmind.com/material/grid](url)
```swift
class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let container = View()
        container.shapePreset = .square
        container.grid.interimSpacePreset = .interimSpace3
        
        view.layout(container).top(100).left(20).right(20)
        
        for _ in 0..<12 {
            let v = View()
            v.grid.columns = 1
            v.backgroundColor = Color.blue.base
            container.grid.views.append(v)
        }

    }

}
```

But only a blank page was showed on my screen. I saw this in Xcode8 by UI View Hierarchy:
![image](https://cloud.githubusercontent.com/assets/10205466/20661360/60e4dc70-b589-11e6-843f-898603ec0443.png)

It seems that the layout is broken.

**Comment by danieldahan** [comment]
Yes, there is an issue. I am pushing a fix within the hour. Thank you!

**Comment by danieldahan** [comment]
Please update to [Material 2.3.20](https://github.com/CosmicMind/Material/releases/tag/2.3.20), where you can find a fix for this issue. :) Thanks

**Comment by samma89** [comment]
Hi,

It seems this issue is there in 3.1.6 release. 
I am running your sample grid from Samples [repo](https://github.com/CosmicMind/Samples/blob/development/Projects/Programmatic/Grid/Grid/ViewController.swift).

![Screen Shot 2019-08-27 at 7 25 21 AM](https://user-images.githubusercontent.com/2077773/63735021-10927d80-c89c-11e9-8b02-aff0574745f7.png)

Any idea of why?

Thanks

**Comment by samma89** [comment]
@danieldahan Thank you!

**Comment by danieldahan** [comment]
@yunhao Hi there, sorry for any delays. We discovered that some earlier updates had caused a regression. You can find a fix in [Material 3.1.7](https://github.com/CosmicMind/Material/releases/tag/3.1.7). Please test, and if there is still an issue, please reopen this issue or create a new issue. Thank you!
